.circleci/config.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       - run: cargo x lint
       - run: cargo xclippy --workspace --all-targets
       - run: cargo xfmt --check
+      # TODO(joshlind): turn this on for all workspace dependencies! We only do one now as a proof of concept.
+      - run: cargo sort --grouped --check state-sync/aptos-data-client/
```

Comment:
We should just turn this on for all of them, but I'm going to be selfish and say that my dependency PR has to go through first :P

Reply:
Yeah, this is the exact reason -- it'll break so many people's PRs 😄 I figure I'll roll it out.